supply-chain-security/compromises/compromise-definitions.md

```diff
@@ -55,5 +55,15 @@ _Reference(s)_:
 - [Wikipedia Code Signing](https://en.wikipedia.org/wiki/Code_signing)
 
 
+## Malicious Maintainer
+Occurs when a maintainer deliberately injects a vulnerability somewhere in the supply chain or in the source code.
```

Comment:
I think we want to capture people posing as maintainers (i.e. compromised a maintainer's account) in this definition too? Maybe

```suggestion
Occurs when a maintainer, or an entity posing as a maintainer, deliberately injects a vulnerability somewhere in the supply chain or in the source code.
```

Reply:
Makes sense.
Also, it could be hard to trace if the motive behind the maintainer actions, because even if the motives were bad initially, then
tomorrow he/she can easily say: "my account was hacked".